vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -1,3 +1,177 @@
+|`ent bundle init [name] --from-hub`| Initialize a bundle from an Entando Hub
+
+#### Command Details
+* `ent bundle init [name] --from-hub`: Utilize an existing bundle from an Entando Hub to jump start your project. The ent bundle CLI will pull the package and rebuild the structure as needed, ready for customization. 
+
+## Build
+ 
+| Command| Descriptions
+|:--|:--
+|`ent bundle build [component-name]` | Build a single component
+|`ent bundle build [mfe-1] [ms-2]...`| Build every the component passed as arguments
+|`ent bundle build --all`|Build all the components in the project
+|`ent bundle build --all-ms`|Build all microservices|
+|`ent bundle build --all-mfe`|Build all the microfrontend|
+
+#### Command Details
+`ent bundle build`: Constructs the project files based on the entando.json provided. It executes according to the type of stack your components are built with. For instance, npm build will be performed for a React MFE. A log file is generated for each component inside the .entando/logs directory for your project.
+
+## Run	
+| Command| Descriptions
+|:--|:--
+|`ent bundle run [component-name]` | Locally run your single component
+|`ent bundle run [mfe-1] [ms-2]...`| Locally run all the components passed as arguments 
+|`ent bundle run --all`| Locally run all the components in the bundle
+|`ent bundle run --all-ms`| Locally run the MSs in the bundle
+|`ent bundle run --all-mfe`| Locally run the MFEs in the bundle
+
+#### Command Details
+`ent bundle run`: Executes processes dependent on the component type and stack. For example, MVN spring:boot will execute for a Sping Boot microservice. All the components in the bundle will run in parallel, with a log file generated for each inside the .entando/logs directory.
+
+## Package
+| Command| Descriptions
+|:--|:--
+|`ent bundle pack`|Generate the bundle artifacts, and the bundle Docker image and MS images 
+|`ent bundle pack --org [organization]`|Generate the bundle artifacts, passing the organization name to the Docker Hub|
+|`ent bundle pack --file [my-dockerfile]`|Use a custom Dockerfile for your bundle |
+
+#### Command Details
+* `ent bundle pack`: For the MFEs and MSs, the generated artifacts are located inside each component's folders. Their format depends on the component type.  For instance, a React micro frontend may result in HTML, JavaScript and CSS files.  
+
+When artifact generation is complete, microservices Docker images are built, executing the docker build command using the Dockerfile located in each microservice folder. If the Dockerfile is missing, the pack command exits with failure.
+
+** Should we include info about where yaml descriptors are located, how they are produced from entando.json, pom.xml or package.json, or how to see the outputs for the MFE in the DEBUG mode
+
+## Publish
+| Command| Descriptions
+|:--|:--
+|ent bundle publish| Publish your Docker images to index.docker.io (default)
```

Comment:
Let's just say the Docker Registry here rather than tie ourselves to a specific URL, especially since I think index.docker.io is the old deprecated URL.